equipment-models/src/main/java/com/telecominfraproject/wlan/equipment/models/events/EquipmentChangedEvent.java

```diff
@@ -10,9 +13,20 @@
  */
 public class EquipmentChangedEvent extends EquipmentEventWithPayload<Equipment> implements HasLocationId {
     private static final long serialVersionUID = 7142209997917559985L;
+    private EquipmentChangeType equipmentChangeType;
+    private Map<RadioType, Integer> newPrimaryChannels;
+    private Map<RadioType, Integer> newBackupChannels;
 
     public EquipmentChangedEvent(Equipment equipment){
         super(equipment.getCustomerId(), equipment.getId(), equipment.getLastModifiedTimestamp(), equipment);
+        equipmentChangeType = EquipmentChangeType.All;
+    }
+    
+    public EquipmentChangedEvent(Equipment equipment, Map<RadioType, Integer> primaryChannels, Map<RadioType, Integer> backupChannels){
```

Comment:
If EquipmentChannelChangedEvent is extended from EquipmentChangedEvent then the extra attributes for newPrimaryChannels and newBackupChannels could be members of EquipmentChannelChangedEvent. In this way, EquipmentChangedEvent would be clean for EquipmentChangeType.All

Reply:
@christieunitt-cu , I think you are still reviewing the old PR. Please see new EquipmentChannelChangedEvent.

Reply:
I think that's a good idea, Christie. This would also help when more incremental-change events are introduced (like for cell-size adjustments, etc.) - the code would be cleaner for each of those new events to extend the EquipmentChangedEvent with some extra attributes.

Reply:
> @christieunitt-cu , I think you are still reviewing the old PR. Please see new EquipmentChannelChangedEvent.

My bad - thanks Lynn. The changes look great!